src/components/LoginForm.jsx

```diff
@@ -91,10 +92,10 @@ const LoginForm = props => {
     const fetchAuthProviders = async () => {
       try {
         const response = await axios.get(`${server}/auth/providers`);
-        setOidcAuthConfig(response.data.filter(config => Object.keys(config).includes('oidc')).map(config => {
+        setOAuthConfig(response.data.filter(config => config.oauth2 !== null).map(config => {
```

Comment:
```suggestion
        setOAuthConfig(response.data.filter(config => config.oauth2 != null).map(config => {
```
Accessing key from object that does not exist returns `undefined` not `null`